src/languages/twig.js

```diff
@@ -27,17 +27,17 @@ function(hljs) {
   var FILTER = {
     begin: /\|[A-Za-z_]+:?/,
     keywords:
-      'abs batch capitalize convert_encoding date date_modify default ' +
-      'escape first format join json_encode keys last length lower ' +
-      'merge nl2br number_format raw replace reverse round slice sort split ' +
-      'striptags title trim upper url_encode',
+      'abs batch capitalize column convert_encoding date date_modify default ' +
+      'escape filter first format inky inline_css join json_encode keys last ' +
```

Comment:
now inky_to_html

Reply:
And does inky stay as deprecated?

Reply:
Good point. We could do either: inky *is* deprecated... but it was from such a new feature, that it only existed for some months before being deprecated. Having it or removing it - both are probably fine.